src/components/Grid.re

```diff
@@ -0,0 +1,35 @@
+open Css;
+
+let gridContainer = (spacing: int) => [
+  width(`calc((`add, pct(100.0), px(spacing)))),
+  margin(px(- spacing / 2)),
+];
+
+let gridItem = (spacing: int) => [padding(px(spacing / 2))];
+
+let component = ReasonReact.statelessComponent("Grid");
+
+type gridType =
+  | Container
+  | Item;
+
+let make = (~type_: gridType, children) => {
+  let getGridStyles = (): list(rule) => {
+    switch (type_) {
+    | Container =>
+      [display(`flex)]
+      @ gridContainer(Theme.spacingPx.small)
+      @ [media(Breakpoints.up(Md), gridContainer(Theme.spacingPx.medium))]
```

Comment:
`gridContainer()` returns an array, right? So is it an array inside of an array?

Reply:
Yeah, when you want to use `media` you need to send in an array of styles that will be applied to that breakpoint, so eventually you will have an outer array with styles not applied to any breakpoints and breakpoints where each of them has its internal array of styles.